.github/components.yml

```diff
@@ -2,6 +2,7 @@ Core:
   revalidate: 'all'
 
 transformations:
+  # preliminary, we need to run all
```

Comment:
it looks like C API and JS API also run some inference tests or at least model compilation tests, so the transformations might affect them and we can't exclude them.

Reply:
> revalidate: 'all'

means that all tests for all other components are run. It means C and JS APIs tests are also run